Dockerfile

```diff
@@ -45,20 +46,29 @@ RUN ollama serve & while ! curl http://localhost:11434; do sleep 1; done; ollama
 # Setup the custom API and frontend
 WORKDIR /workspace
 COPY --chmod=644 gswikichat gswikichat
-COPY --chmod=755 frontend frontend
+COPY --chmod=755 static static
```

Comment:
Did we decide to go back to calling the frontend "static", even for Vue3?

Reply:
That must have been a merge error. Sorry for that.